packages/editor/src/client/utility.ts

```diff
@@ -127,3 +127,17 @@ export function transformCssStringToObject(styleCustom: string): Record<string,
     return previousValue
   }, {})
 }
+
+export function useStickyState(
```

Comment:
Don't put this into `utility.ts` it's used for saving current language to localStorage so you should put it where you need it.